docs/snippets/package/azure/composition.yaml

```diff
@@ -30,6 +30,8 @@ spec:
             location: West US 2
             sslEnforcement: Disabled
             version: "11"
+            storageProfile:
+              storageMB: 5120            
```

Comment:
This was a small bug that I found. The user will pass in the storage capacity in GB, but we do a multiplication transform in the patch.